src/routes/SignUp.jsx

```diff
@@ -0,0 +1,155 @@
+import React, { useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link, Navigate } from 'react-router-dom';
+import { toast } from 'react-toastify';
+import * as Yup from 'yup';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { selectAuth } from '../redux/store';
+import { registerUser } from '../redux/auth/authActions';
+import useSession from '../hooks/useSession';
+import Input from '../components/Input';
+import Button from '../components/Button';
+
+const SignUp = () => {
```

Comment:
- [ ] When I try to use the provided admin credentials, I get this error:

![image](https://github.com/Atril33/DocConnect-Frontend/assets/57421823/2d7d5d6e-742c-425d-b4da-b166c5c2e498)

Kindly, take a look at this issue so the "add" and "delete" can be accessed.

note: This may be working on your end because your browser's local storage is already populated, which is not the case for new users.  In order to correctly debug this, kindly test it on an incognito tab.

note 2: This means I couldn't get into the admin functionality.  This will be taken a look in the next review.